src/index.html

```diff
@@ -1,13 +1,49 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;900&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <img
+        class ="product__image"
+        src="images/imac.jpeg"
+        alt="product-image"
+      >
+      <p class="product__description">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="product__code">
+        Код товара: 195434
+      </p>
+      <div class="product__rating">
+        <div class="product__stars">
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+        </div>
+        <p class="product__feedbacks">
+          Отзывов: 5
+        </p>
+      </div>
+      <div class="product__value">
+        <p class="product__caption">Цена:</p>
+        <p class="product__price">69 999 грн</p>
```

Comment:
change p to span, this is not paragraph's

Reply:
changed. Good advice. thanks